std/file.d

```diff
@@ -2737,17 +2748,21 @@ bool attrIsSymlink(uint attributes) @safe pure nothrow @nogc
 
     auto source = deleteme ~ "source";
     auto target = deleteme ~ "target";
-    scope(exit) source.remove, target.remove;
 
     assert(!source.exists);
     assertThrown!FileException(source.getLinkAttributes.attrIsSymlink);
 
-    target.write("target");
-    target.symlink(source);
-    assert(source.readText == "target");
-    assert(source.isSymlink);
-    assert(source.getLinkAttributes.attrIsSymlink);
-    assert(!source.getAttributes.attrIsSymlink);
+    // symlinking isn't available on Windows
```

Comment:
This does not look good in an example of `attrIsSymlink`, which *is* available on Windows.

Reply:
It's a hotfix to get auto-tester passing again (the examples were added today with #6320, so we still have quite some time to improve them until they are part of the released docs).

Anyhow, I didn't know a better way to show off `attrIsSymlink` in a cross-platform way.
Just `version(Posix) unittest` isn't a good idea either, because then code on dlang.org doesn't work when copy/pasting locally